components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -35,41 +36,40 @@ class ToolbarPresenter(
      */
     fun stop() {
         sessionManager.unregister(this)
-        session.unregister(this)
+        activeSession.unregister(this)
     }
 
     /**
      * A new session has been selected: Update toolbar to display data of new session.
      */
     override fun onSessionSelected(session: Session) {
-        this.session.unregister(this)
+        activeSession.unregister(this)
 
-        this.session = session
+        activeSession = session
         session.register(this)
 
```

Comment:
Do we need to check here whether we were started with a specific session and then ignore selection changes? Most of the time this won't be a problem as the selection won't change while the custom tab activity is in the foreground - but you may be able to have both activities visible using Android's split screen feature.

Or should we just not register an observer on the session manager in onStart() if we got a session? We only listen for a selection change right now.